src/models/transactions/common.ts

```diff
@@ -29,7 +29,13 @@ const transactionTypes = [
 
 const MEMO_SIZE = 3
 
-function isMemo(obj: { Memo?: unknown }): boolean {
+/**
+ * Exported for testing only.
+ *
+ * @param obj - An unknown object.
+ * @returns Boolean if the object is a memo.
+ */
+export function isMemo(obj: { Memo?: unknown }): boolean {
```

Comment:
this can be tested via `verifyBaseTransaction` (just add some memos there), so you don't need to export it.